devise.rb

```diff
@@ -4,23 +4,24 @@
 ########################################
 inject_into_file 'Gemfile', before: 'group :development, :test do' do
   <<~RUBY
-    gem 'devise'
-
     gem 'autoprefixer-rails', '10.2.5'
+    gem 'devise'
     gem 'font-awesome-sass'
     gem 'simple_form'
   RUBY
 end
 
 inject_into_file 'Gemfile', after: 'group :development, :test do' do
   <<-RUBY
+
+  gem 'dotenv-rails'
   gem 'pry-byebug'
   gem 'pry-rails'
```

Comment:
Since we now have the debug gem in the group, 
![image](https://user-images.githubusercontent.com/57643651/156160804-a3d924c1-65dc-4ae3-b9f3-e59355fa745d.png)

Should we do instead:

```suggestion
inject_into_file 'Gemfile', after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ] ' do
  <<-RUBY
  gem 'dotenv-rails'
  gem 'pry-byebug'
  gem 'pry-rails'
```

For them to be in the right order too? (same for minimal) Can do the change if you agree 😊